### ISSUE TITLE
NPE in TravelTimeReducer: travelTimeResult is null

### ISSUE THREAD

**Issue by trevorgerhardt** [conveyal/r5#679]
When running a regional analysis after #665 was merged I get an NPE. This line appears to be the culprit: https://github.com/conveyal/r5/pull/665/files#diff-98772725b2b1ffbbef0b035f1bed2f7d91fd116f2b1afe72aa296323530a95d4R205

A few questions:
1. Should we be checking the `calculateTravelTimes` flag or more direclty if `travelTimeResult` is `null` before calling that method?
2. Should we default `recordTimes = true` on a `RegionalTask`? 
3. Is there a UI change that was supposed to occur simultaneously that I was unaware of?

**Comment by abyrd** [comment]
Thanks for the link to the offending line. This is indeed just missing a null check - I must not have tested cases without travel times. To hit all your questions:

1. I believe `calculateTravelTimes` directly implies `travelTimeResult != null` so the checks are equivalent. Although `calculateTravelTimes` seems more descriptive, here I prefer to check `travelTimeResult != null` because it so simply connects with the action of calling a method on `travelTimeResult` on the next line.
2. No, recording times on regional tasks could produce huge amounts of information possibly overwhelming the backend (travel times to millions of destinations returned from every origin), so we only want to enable it in special cases where we know the number of destinations are under control
3. No UI changes were expected. What was added here is the histogram recording, which we don't (yet?) expose in the UI. So far it's only used in tests to check that detailed travel times to certain destinations match a theoretically predicted distribution. It's equivalent to the whisker plots we're showing over the map for travel time to one destination, but more detailed.

In short this is only intended to be enabled in debugging/testing situations, and just needs a null check. I will fix this now.
